setup.cfg

```diff
@@ -55,8 +55,7 @@ install_requires =
     tables
     pyzmq
     moviepy
-    imageio
-    imageio-ffmpeg; python_version > "2.7"
+    imageio <= 2.4.1
```

Comment:
This new line should be removed from `master`.

Reply:
Yup. Done in my master